CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Added
+
+-   Implement `ProposalVoteDataListItem` with Structure and Skeleton module
```

Comment:
Last feedback from Selim was to call this component `VoteProposalDataListItem` instead of `ProposalVoteDataListItem`